pages/common/ln.md

```diff
@@ -6,6 +6,10 @@
 
 `ln -s {{path/to/file}} {{path/to/symlink}}`
 
+- Create several symbolic links to one file (or folder):
+
+`ln -s {{path/to/file}} {{path/to/symlink1}}`
```

Comment:
Sorry, maybe I am misunderstanding something. Isn't this exactly the same as the previous example ? You seem to have just changed `symlink` to `symlink1`.

Reply:
@agnivade Sorry, you're right, my bad. I just forgot the additional symlinks. I pushed the fix to my branch. Should be fine now.